CHANGELOG.md

```diff
@@ -21,7 +27,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [0.0.1] - 2015-02-18
 
-[Unreleased]: https://github.com/conjurinc/memtar/compare/v0.1.3...HEAD
+[Unreleased]: https://github.com/conjurinc/memtar/compare/v0.1.4...HEAD
+[0.1.4]: https://github.com/conjurinc/memtar/compare/v0.1.1...v0.1.4
```

Comment:
It's not just yours, because I notice the ones below are wrong too... :-/

Would you mind correcting the others as well?

```suggestion
[0.1.4]: https://github.com/conjurinc/memtar/compare/v0.1.3...v0.1.4
```